### ISSUE TITLE
Bootstrap application with routing and placeholder pages

### ISSUE THREAD

**Issue by thomaseizinger** [comit-network/comit-i#8]
As a starting point for the remaining tasks, we should bootstrap the application with the tech decided in #5 and create routes for the pages described in #2:

1. A page for listing all swaps
2. A page for creating a new swap link
3. A landing page for "swap links"

The application should also have a navigation bar for navigating to the following pages:

- List all swaps (1)
- Create new swap link (2)

Pages for actions are out of scope because of #12.

Blocked by #5.
Child of #3.
Replaces #4.
